lib/main.coffee

```diff
@@ -351,10 +351,10 @@ module.exports = Hydrogen =
             @wsKernelPicker = new WSKernelPicker (kernel) =>
                 @clearResultBubbles()
 
-                grammar = @editor.getGrammar()
+                grammar = kernel.grammar
```

Comment:
I triggered an inconsistency like @nikitakit mentioned.
To trigger it, select a remote kernel (preferably over a poor internet connection) and quickly switch to an other tab with different grammar. `@editor.getGrammar()` and `kernel.grammar` will not be the same (I checked for equality at line 355).